### DISCUSSION TITLE
change default username on SMTP Credentials

### DISCUSSION THREAD

**Discussion by lvaro22** [postalserver/postal#2617] [Help with using Postal]
Hello,

If I wish to alter the default username while creating SMTP credentials, what steps should I take ?
I have to do this because the username field in a different program doesn't allow the use of a slash symbol.

Thanks you for the attention.

**Comment by willpower232** [comment]
Postal doesn't actually use the username, it is mostly present for convention. Have a look at #1498

**Comment by lvaro22** [comment]
Ok, thank you!
